python/lsst/afw/geom/angle/angleContinued.py

```diff
@@ -0,0 +1,54 @@
+#
+# LSST Data Management System
+# Copyright 2008-2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import, division, print_function
+
+__all__ = []
+
+from lsst.utils import continueClass
+from .angle import Angle, AngleUnit, radians
+
+
+@continueClass
+class Angle:
+
+    def __abs__(self):
+        return abs(self.asRadians())*radians
+
+    def __reduce__(self):
+        return (Angle, (self.asRadians(),))
+
+
+@continueClass
+class AngleUnit:
+
+    def __mul__(self, other):
+        if isinstance(other, (Angle, AngleUnit)):
+            raise NotImplementedError
+        return AngleUnit._mul(self, other)
```

Comment:
I'm really not sure; this is code I moved but did not write.  Since it seems you have ambitions to clean up Angle's maddeningly inconsistent set of operators, I'm hoping you'll be able to fix this later.